docker-images/notebook/Dockerfile

```diff
@@ -3,76 +3,81 @@ FROM jupyter/base-notebook:4417b81d04b7
 
 USER root
 RUN apt-get update \
-  && apt-get install -yq --no-install-recommends libfuse-dev nano fuse vim git graphviz \
+  && apt-get install -yq --no-install-recommends libfuse-dev nano fuse vim git \
```

Comment:
You can install `git`, `nano` from `conda-forge` too. (`vim` and `fuse` will be available soon.)

Reply:
Any advantage in doing this?